packages/devextreme/js/__internal/grids/grid_core/columns_resizing_reordering/m_columns_resizing_reordering.ts

```diff
@@ -1054,15 +1131,28 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
       }
       return this._pointsByColumns;
     }
-  },
+  }
 
   dispose() {
     this._unsubscribes();
-    this.callBase();
-  },
-});
+    super.dispose();
+  }
+}
+
+class TablePositionViewController extends modules.ViewController {
+  private _columnHeadersView: any;
+
+  private _rowsView: any;
+
+  private readonly positionChanged: any;
```

Comment:
Minor comment.

We can do inline initialization of the field here:
```typescript
private readonly positionChanged = Callbacks();
```

Reply:
It's dangerous cause constructor call may be skipped